apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -16,61 +15,41 @@
 import gov.cms.bfd.model.rif.SkippedRifRecord;
 import gov.cms.bfd.model.rif.SkippedRifRecord.SkipReasonCode;
 import gov.cms.bfd.model.rif.entities.Beneficiary;
-import gov.cms.bfd.model.rif.entities.BeneficiaryCsvWriter;
 import gov.cms.bfd.model.rif.entities.BeneficiaryHistory;
 import gov.cms.bfd.model.rif.entities.BeneficiaryMonthly;
 import gov.cms.bfd.model.rif.entities.Beneficiary_;
-import gov.cms.bfd.model.rif.entities.CarrierClaim;
-import gov.cms.bfd.model.rif.entities.CarrierClaimCsvWriter;
-import gov.cms.bfd.model.rif.entities.CarrierClaimLine;
 import gov.cms.bfd.model.rif.parse.RifParsingUtils;
 import gov.cms.bfd.pipeline.ccw.rif.load.RifRecordLoadResult.LoadAction;
 import gov.cms.bfd.pipeline.sharedutils.IdHasher;
 import gov.cms.bfd.pipeline.sharedutils.PipelineApplicationState;
 import gov.cms.bfd.pipeline.sharedutils.TransactionManager;
 import gov.cms.bfd.sharedutils.exceptions.BadCodeMonkeyException;
-import java.io.File;
-import java.io.FileReader;
-import java.io.FileWriter;
 import java.io.IOException;
-import java.io.UncheckedIOException;
 import java.math.BigDecimal;
-import java.sql.Connection;
-import java.sql.SQLException;
 import java.time.Instant;
 import java.time.LocalDate;
 import java.time.Period;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Comparator;
 import java.util.LinkedList;
 import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Consumer;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import javax.persistence.Entity;
 import javax.persistence.EntityManager;
-import javax.persistence.EntityManagerFactory;
 import javax.persistence.EntityTransaction;
-import javax.persistence.Table;
 import javax.persistence.criteria.CriteriaBuilder;
 import javax.persistence.criteria.CriteriaQuery;
 import javax.persistence.criteria.JoinType;
 import javax.persistence.criteria.Root;
-import org.apache.commons.csv.CSVFormat;
 import org.apache.commons.csv.CSVPrinter;
```

Comment:
Does that whole section that starts on line 439 with  `if (!isSyntheticData && isBackdatedBene(rifRecordEvent)) { ` have to go?

Reply:
so lines 443-452 can go; that section is basically handling 'skipped records'; that logic effectively checks that the data is 'not syntheticData' (i.e., so real production data). So if production data AND that data arrives with a ref_yr mismatch, the ETL 'skips' those records but does write them to the db (skipped_rif_records table). so we need to preserve lines 454 on). But we don't need to preserve a skipped record as a csv.

Reply:
Do you mean the record would be written without the CSV string?  Just leav `rifData` null or empty?

Reply:
We write 'skipped' records into the db in that section; we would not need a separate RIF file of 'skipped' data at all here. Not sure if RDA (PACA) treats it differently tho' ???

Reply:
OOOPPPPSSS!!!

I take it all back...we do need this section...forgot that the skipped RIF rcd in the db includes the raw content!

Ig nore all my previous notes...I am a moron and not enough coffee!

Reply:
But the RIF data IS the record we are writing.  The other columns aren't the data, just some meta data to go along with the RIF.  This may be a great thing to change but it isn't really in scope for this store.

Reply:
Reverted the skipped record changes in RifLoader